### PR TITLE
Update ffi utils

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: safe_app/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,13 @@
 - Fix development build links in release description
 - Add missing iOS builds to GHA deploy + fix tagging for releases
 
+## [0.13.0]
+- Update to safe-nd 0.7.2
```
Comment (bochaco) [inline safe_app/CHANGELOG.md]: We are in version 8 already

File: safe_authenticator/CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,13 @@
 - Fix development build links in release description
 - Add missing iOS builds to GHA deploy + fix tagging for releases
 
+## [0.13.0]
+- Update to safe-nd 0.7.2
```
Comment (bochaco) [inline safe_authenticator/CHANGELOG.md]: We are in version 8

File: safe_core/CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,16 @@
 - Make returned error codes to be positive numbers
 - Remove pedantic warnings
 
+## [0.38.1]
+- Fix broken master workflow
+
+## [0.38.0]
+- Update to safe-nd 0.7.2
```
Comment (bochaco) [inline safe_core/CHANGELOG.md]: We are in version 8

File: safe_app/CHANGELOG.md (hunk 1 above)
Comment (joshuef) [reply safe_app/CHANGELOG.md]: that's a missing version 13 changelog

File: safe_authenticator/CHANGELOG.md (hunk 2 above)
Comment (joshuef) [reply safe_authenticator/CHANGELOG.md]: same. this changelog for this version was missing. this came up in a rebase, so I figured: keep em?

**Comment by bochaco** [comment]
Ah right @joshuef , I'm sorry, we are even missing the changelog for release 0.14, yes we need to keep it the changelog ofc

**Pull request merged**